src/topography.jl

```diff
@@ -1,6 +1,16 @@
-#####
-##### Topography
-#####
+import ArtifactWrappers as AW
+
+function topo_elev_dataset_path()
+    etopo1_elev_data = AW.ArtifactWrapper(
+        @__DIR__,
+        "topo-elev-info",
+        AW.ArtifactFile[AW.ArtifactFile(
+            url = "https://caltech.box.com/shared/static/gvilybsu5avxso1wubxjthpip9skc6mf.nc",
+            filename = "ETOPO1_coarse.nc",
+        ),],
+    )
+    return AW.get_data_folder(etopo1_elev_data)
+end
```

Comment:
Can we move this to `examples/` and remove the dependence on `ArtifactWrappers`?